useid-eservice-sdk/src/IdentityData.ts

```diff
@@ -0,0 +1,13 @@
+import { DataGroup } from "./DataGroup";
+
+export interface IIdentityDataValues {
+  [key: string]: string | undefined;
+}
+
+export class IdentityData {
+  constructor(private values: IIdentityDataValues) { }
+
+  get(dataGroup: DataGroup): string | undefined {
```

Comment:
I don't know much about typescript, but is `undefined` a defined return type? so in case of non existing dataGroup it will return `undefined`?

Reply:
Yes, exactly. That's the TypeScript way of defining "nullables".